libnymea-core/servermanager.cpp

```diff
@@ -370,15 +369,26 @@ void ServerManager::setServerName(const QString &serverName)
     qCDebug(dcServerManager()) << "Server name changed" << serverName;
 
     foreach (WebSocketServer *websocketServer, m_webSocketServers.values()) {
+        unregisterZeroConfService(websocketServer->configuration().id, "ws");
         websocketServer->setServerName(serverName);
+        registerZeroConfService(websocketServer->configuration(), "ws", "_ws._tcp");
```

Comment:
Is unregister `configuration().id` and register `configuration()` correct?

Reply:
yeah... for registering we need all the info, which ports etc... For unregistering, the id is enough... I know, api-wise it'd be nicer to have it aligned, however, it would also require the caller to fetch the entire info just for unregistering which isn't really required, so unregistering can be done with the id alone.